orgsearch.html

```diff
@@ -1,27 +1,122 @@
-    </div>
-    <div>
-        <a href="petsearch.html" role="menuitem" class="navigation-list_item">Search by Pet</a>
-    </div>
-
-
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+        <link rel="stylesheet" href="styles.css">
+        <title>Document</title>
+    </head>
+    <body style="background-color: rosybrown;"></body>
+        <h1 style="text-align: center; color:rgb(223, 175, 175)">Find Your New Best Friend</h1>
+        <header class="bg-light text-center mb-4 p-3">
+            <img src="adopt.jpg" alt="Find-A-Pet">
+        </header>
+        <div class="container">
+            <div>To Search by Organizations Near You:</div>
+            <fieldset>
+                <form class = "row me-3 mb-3" id="pet-form">
+                    <div class = "col-sm-6">
+                        <input type="text" id="zip" class="form-control form-control-lg" placeholder="Zipcode" required pattern=[0-9]{5}>
+                    </div>
+                    <div class = "col-sm-2">
+                        <select class="form-select form-select-lg" aria-label="Default select example" id="distance">                            
+                            <option selected disabled value>Distance</option>
+                            <option value="1">10 Miles</option>
+                            <option value="2">25 Miles</option>
+                            <option value="3">50 Miles</option>
+                            <option value="4">100 Miles</option>
+                        </select>
+                    </div>
+                    <div class = "col-sm-2">
+                        <button class="btn btn-primary btn-lg btn-block" type ="submit">Submit</button>                    
+                    </div>
+                </form>
+                <div>OR</div>
+                <form class = "row me-3 mb-3" id="pet-form">
+                    <div class = "col-sm-3">
+                        <input for="validationDefault03" type="text" id="city" class="form-control form-control-lg" placeholder="City" required>
+                    </div>
+                    <div class = "col-sm-3">
+                        <select class="form-select form-select-lg" id="state" name="state" aria-label="default select example" required>
+                            <option selected disabled value="">State</option>
+                            <option value="AK">Alaska</option>
```

Comment:
I think you are missing of the some states, and if we are going to list them we could put them in alphabetical order. Here's a list I found: https://alphabetizer.flap.tv/lists/list-of-states-in-alphabetical-order.php

I don't know if Puerto Rico or District of Columbia comes out for the API, but it's worth checking! I can take a look tomorrow.

Reply:
So I doubled checked the API

-  It CAN call District of Columbia as "DC" for Location.
- But the API doesn't understanding calling Puerto Rico as a state, however you can search for Zip Codes within Puerto Rico. Maybe we take off Puerto Rico as a state call and let people call by Zip Codes?